templates/react-native-template-realm-ts/README.md

```diff
@@ -20,11 +23,80 @@ npm run ios
 ```
 npm run android
 ```
+
 ## 💻 Start the Dev Client
 
 ```
 npm start
 ```
+
+## 💾 Testing changes to the template when developing
+
+To test the template locally, run it like any other React Native app: `npm i && npx pod-install` then `npm run ios`.
+
+## 🔀 Setting up sync
+
+### Setting up a Realm app in the cloud
+
+To enable sync, first you need to register for MongoDB Atlas and set up a new free Realm app:
```

Comment:
Should we even be providing these steps ourselves?  There has got to be documentation on this in our docs pages.  I think we will have a hard time maintaining this if there are any changes to the UI in Atlas.

Reply:
I feel like this might be OK as an interim solution, and we can speak to the docs team about creating something easier to follow (like illustrated, or a video).... and I guess give feedback that the cloud setup experience could be a lot easier for first time users